src/shared/util/numbers.util.ts

```diff
@@ -6,3 +6,56 @@ export const getRandomInt = (min: number, max: number = LARGEST_SAFE_INTEGER) =>
 
 export const getRandomPercentage = () =>
   faker.number.float({ min: 0, max: 100, fractionDigits: 2 })
+
+export const getRandomElementFromArray = (array: any[]) =>
+  array[getRandomInt(0, array.length - 1)]
+
+/**
+ * Formats a phone number for AWS Route53 domain registration
+ * @param phoneNumber - The phone number to format
+ * @param fallbackNumber - Optional fallback number to use if phoneNumber is invalid
+ * @returns Properly formatted phone number in +1.XXXXXXXXXX format
+ */
+export const formatPhoneNumber = (
+  phoneNumber?: string,
+  fallbackNumber?: string,
+): string => {
+  if (!phoneNumber || phoneNumber.trim() === '') {
+    if (fallbackNumber) {
+      return fallbackNumber
+    }
+    throw new Error('No phone number provided and no fallback available')
+  }
+
+  const cleanedNumber = phoneNumber.replace(/\D/g, '')
+
+  if (cleanedNumber === '') {
+    if (fallbackNumber) {
+      return fallbackNumber
+    }
+    throw new Error('Invalid phone number format and no fallback available')
+  }
+
+  if (cleanedNumber.length === 10) {
+    return `+1.${cleanedNumber}`
+  } else if (cleanedNumber.length === 11 && cleanedNumber.startsWith('1')) {
+    return `+1.${cleanedNumber.substring(1)}`
+  } else if (cleanedNumber.length > 11) {
+    // Check if the first 11 digits form a valid US number
+    if (cleanedNumber.startsWith('1')) {
+      return `+1.${cleanedNumber.substring(1, 11)}`
+    } else {
+      // Non-US number or invalid format, use fallback
+      if (fallbackNumber) {
+        return fallbackNumber
+      }
+      throw new Error('Non-US phone number and no fallback available')
+    }
+  } else {
+    // Invalid length, use fallback
+    if (fallbackNumber) {
+      return fallbackNumber
+    }
+    throw new Error('Invalid phone number length and no fallback available')
+  }
+}
```

Comment:
Seems like this util should just replace the other `formatPhoneNumber` we already have in the `AwsModule`, that's specific to the expected format for AWS services, rather than add one here in a `numbers.util.ts` that has two args that are both `string` and also the method returns `string`, not `number` in anycase.

https://github.com/thegoodparty/gp-api/blob/develop/src/aws/util/awsRoute53.util.ts#L2-L14